src/components/PokeCard.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export default function PokeCard({ pokemon }) {
+  return (
+    <div className="poke-card" href={pokemon.pokedex}>
+      <h2>{pokemon.pokemon}</h2>
+      <img src={pokemon.url_image} />
+      <p>
+        types: {pokemon.type_1}, {pokemon.type_2}
+      </p>
+    </div>
```

Comment:
little redundant, if you can refactor, try and destructure your pass in after you spread it in the main.js.